app/templates/events/trackVolunteers.html

```diff
@@ -76,13 +76,25 @@ <h3>{{eventData["program"].programName}} Event</h3>
       </tr>
     </thead>
     <tbody class="align-middle" id="volunteerTable">
+    {% macro rsvpHistoryData(volunteerUser) %}
+      {% if eventVolunteerData %}
+        {% for volunteer in eventVolunteerData %}
+          {% if volunteerUser == volunteer.user %}
+            <p>{{volunteer.user.firstName}}'s last rsvp was on {{volunteer.rsvpTime.strftime('%b %-d %-I:%M %p')}}</p>
+            {% if volunteer.unRsvpTime != None %}
+              <p>{{volunteer.user.firstName}} last unrsvp was on {{volunteer.unRsvpTime.strftime('%b %-d %-I:%M %p')}}</p>
+            {% endif %}
+          {% endif %}
+        {% endfor %}
+      {% endif %}
+    {% endmacro %}
     {% if eventVolunteerData %}
       {% for participant in eventVolunteerData %}
-        <tr>
+        <tr class="{% if participant.unRsvpTime == None or participant.rsvpTime.strftime('%b %-d %-I:%M:%S %p') > participant.unRsvpTime.strftime('%b %-d %-I:%M:%S %p') %}" "{% elif participant.rsvpTime.strftime('%b %-d %-I:%M:%S %p') < participant.unRsvpTime.strftime('%b %-d %-I:%M:%S %p')%}table-secondary{% endif %}" data-toggle="popover">
```

Comment:
I think it would be better to set a variable to the results of these checks and then pass that variable into the tr rather than doing it all in the tr

Reply:
fixed!